src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,15 @@ public class User {
     private String password;
     private Integer age;
 
+    public User() {
```

Comment:
for my solution i didn't change model by default for  adding furthermore methods